### PR TITLE
feat: Highlighting

### PR DESCRIPTION
**Pull request by j03-dev**
There's a pull request for this, but I don't know what else to highlight.

### CODE REVIEW

File: src/config/color.rs [hunk 1]
```diff
@@ -78,19 +78,44 @@ impl<'b> Colorizer<'b> {
     pub fn patch_text<'a>(&self, str: &'a str) -> Vec<Line<'a>> {
         let text = str.into_text().unwrap().patch_style(self.base_style);
 
-        if self.colors.is_empty() {
-            // We don't have color options.
-            // Just return base-styled lines.
-            return text.lines;
-        }
+        let mut color_options = vec![
```
Comment (Natoandro) [inline src/config/color.rs:81]: Can we skip re-creating this `Vec` on each call?

**Review by Natoandro — COMMENTED** [review]
Great!

File: src/config/color.rs (hunk 1 above)
Comment (j03-dev) [reply src/config/color.rs]: I have now included it in lazy_static.

File: src/config/color.rs [hunk 2]
```diff
@@ -78,19 +109,15 @@ impl<'b> Colorizer<'b> {
     pub fn patch_text<'a>(&self, str: &'a str) -> Vec<Line<'a>> {
         let text = str.into_text().unwrap().patch_style(self.base_style);
 
-        if self.colors.is_empty() {
-            // We don't have color options.
-            // Just return base-styled lines.
-            return text.lines;
-        }
+        let mut colors = self.colors.to_owned();
+        colors.extend_from_slice(&COLOR_OPTIONS);
```
Comment (Natoandro) [inline src/config/color.rs:113]: What do you think of [`chain`](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.chain)?

**Review by Natoandro — APPROVED** [review]

File: src/config/color.rs (hunk 2 above)
Comment (j03-dev) [reply src/config/color.rs]: I've never seen this before, but I did some research and found which one works better. (stack_overflow)[https://stackoverflow.com/questions/40792801/what-is-the-best-way-to-concatenate-vectors-in-rust]

**Pull request merged**